source/command-monitoring/command-monitoring.rst

```diff
@@ -150,6 +150,13 @@ assert these conversions take place.
    * - ``OP_DELETE``
      - delete command
 
+Read Preference
```

Comment:
SPEC-35 is specifically about commands wrapped in $query, not query filters.  The text should be generalized so that is applies to commands as well.